### ISSUE TITLE
Specific Spectre crashes PoB (Tentacle Miscreation)

### ISSUE THREAD

**Issue by Valerate** [Openarl/PathOfBuilding#1617]
Hi

I usually do tentacle miscreation summoner, and has never had issue with PoB. But with the latest version it crashes if I select it as my choice for Raise Spectre gem.

This is the Error:

https://i.imgur.com/Oe3U0W2.png

**Comment by DrewCD** [comment]
I can confirm this issue is still present in 1.4.146

**Comment by Kino8** [comment]
Was about to post the same issue. Image of crash screen: https://i.imgur.com/ezEiMAM.png

1.4.146

**Comment by Kino8** [comment]
Considering the OP is 3+ months old, is PoB reliable at this point?

**Comment by Valerate** [comment]
Specters have always been a low priority, both for GGG and... well the community. Especially non-meta one's like this one.

**Comment by Xallionx** [comment]
I imagine that spectres (even off-meta) will get some love with 3.8.
The program crashes because the Tentacle Micreations skill is mislabeled in 3_0\Spectres.lua.  The skill has to exist in 3_0\skills\Spectre.lua

While we wait for this to be officially fixed, you can manually fix the issue with Tentacle Miscreations so you can calculate in POB again.
1) Open C:\ProgramData\Path of Building\Data\3_0\Spectres.lua in a text editor
2) Search for "Tentacle Miscreation"
3) Update the "skillList" to only have the following skill (it currently has 6):
	skillList = {
		"DemonFemaleRangedProjectile",
	},
4) Save and restart PoB.

This fixes the issue.  The only downside is that PoB will always tell you that there is an available update (and when you do update, it will revert this change)

**Comment by Ferhnir** [comment]
> I imagine that spectres (even off-meta) will get some love with 3.8.
> The program crashes because the Tentacle Micreations skill is mislabeled in 3_0\Spectres.lua. The skill has to exist in 3_0\skills\Spectre.lua
> 
> While we wait for this to be officially fixed, you can manually fix the issue with Tentacle Miscreations so you can calculate in POB again.
> 
> 1. Open C:\ProgramData\Path of Building\Data\3_0\Spectres.lua in a text editor
> 2. Search for "Tentacle Miscreation"
> 3. Update the "skillList" to only have the following skill (it currently has 6):
>    skillList = {
>    "DemonFemaleRangedProjectile",
>    },
> 4. Save and restart PoB.
> 
> This fixes the issue. The only downside is that PoB will always tell you that there is an available update (and when you do update, it will revert this change)

Yes, follow those steps and U will fix your Tentacle spectre, gj Xallionx

**Comment by Openarl** [comment]
The error is now fixed as of 1.4.150, however the result wouldn't be to your liking...

This particular monster had its main skill replaced recently (3.7.0, I believe; DemonFemaleRangedProjectile being the original skill). The 2 new skills that replaced it work very differently from most monster skills: they use server-side scripting to repeatedly trigger other skills to achieve a similar effect to the older skill, but with more control from the designer. 

Now, while I personally possess the knowledge required to get these skills working in PoB (partly due to having written the code for the special projectile skills used here), I can't actually use that knowledge, as no one outside GGG would have any real chance of figuring it out themselves (mainly since the scripts aren't included in the client). So unfortunately I can't add support for these skills, and since the old skill is no longer correct for calculating this monster's DPS, I've had to leave this spectre without any skills.
